joomla/joomla-cms
# [4.2] Replace $this->_db usages in components models

Pull Request for Issue # .

### Summary of Changes
This PR `$this->_db` in all components models by database object returned by getDatabase() method. This is not a find and replace PR, so please do careful code review before merging.

### Testing Instructions
1. Use Joomla 4.2
2. Apply patch
3. Access to random pages from both frontend and backend, make sure nothing is broken
4. Please at least one careful code review.
